modules/tests/shared/src/test/scala/lucuma/core/math/EpochSuite.scala

```diff
@@ -150,4 +151,85 @@ final class EpochSuite extends munit.DisciplineSuite {
     // 2020.5 Julian epoch converts to july 2nd assuming a year of 365.25 per definition
     val expected = Instant.parse("2020-07-02T03:00:00Z")
     assertEquals(instant, expected)
+
+  test("formatWithScheme.exactly 2 decimal places"):
+    forAll(ArbEpoch.epochs2Decimal): epoch =>
```

Comment:
Do you really need an Aribitrary that creates Epochs with 2 decimals? Any Epoch should be formatted with the 2 decimals, right?